.github/workflows/coveralls.yml

```diff
@@ -26,22 +26,23 @@ jobs:
         run: python -m pip install --upgrade pip setuptools wheel
       - name: Set up OpenSK
         run: ./setup.sh
+      - name: Install llvm tools
+        run: rustup component add llvm-tools-preview
 
-      - name: Install grcov
+      - name: Install and run grcov
         run: if [[ ! -e ~/.cargo/bin/grcov ]]; then cargo +stable install grcov; fi
       - uses: actions-rs/cargo@v1
         with:
           command: test
           args: --features "with_ctap1,with_nfc,std" --no-fail-fast
         env:
-          CARGO_INCREMENTAL: '0'
-          RUSTFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
-          RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
+          RUSTFLAGS: "-Zinstrument-coverage"
+          LLVM_PROFILE_FILE: "opensk-%p-%m.profraw"
       - uses: actions-rs/grcov@v0.1.5
         id: coverage
       - uses: coverallsapp/github-action@1.1.3
         name: upload report to coveralls
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          path-to-lcov: ${{ steps.coverage.outputs.report }}
```

Comment:
This should be reverted too if you use the github action.

Reply:
No, that was just a test run to confirm that the GitHub Action only works for the gcov based flow. The run failed, and the [documentation](https://github.com/marketplace/actions/rust-grcov) only mentions the former method. The commit now simply expands the parameter names.